foreshadow/transformers/internals/fancyimpute.py

```diff
@@ -15,8 +15,8 @@ class FancyImputer(BaseEstimator, TransformerMixin):
 
     """
 
-    def __init__(self, method="SimpleFill", **kwargs):
```

Comment:
Not super clear on why we switched away from just being able to pass in kwargs (it might make serialization look a little bit more tricky).

Reply:
can't because BaseEstimator get_params() inspects the inits and won't see any variable kwargs. If we pass as explicit kwargs we need to keep our API in line with fancyimputer api changes.